src/probnum/diffeq/ode/__init__.py

```diff
@@ -1,4 +1,15 @@
 from .ivp import *
+from .ivp_examples import *
 from .ode import *
 
-__all__ = ["ODE", "IVP", "logistic", "fitzhughnagumo", "lotkavolterra"]
+__all__ = [
+    "ODE",
+    "IVP",
+    "logistic",
+    "fitzhughnagumo",
+    "seir",
+    "rigidbody",
+    "vanderpol",
+    "threebody",
+    "lotkavolterra",
+]
```

Comment:
Do we want this here? Or is it sufficient to have an __all__ in `diffeq/__init__.py`? I know this was here before, but maybe now is as good as ever to remove it